src/celengine/nebula.cpp

```diff
@@ -22,23 +23,47 @@
 namespace engine = celestia::engine;
 namespace util = celestia::util;
 
+using namespace std::string_view_literals;
 using util::GetLogger;
 
+struct NebulaTypeName
```

Comment:
The definitions of `NebulaTypeName` and `NebulaTypeNames` should be in an unnamed namespace

Reply:
NebulaTypeName, NebulaTypeNames, NebulaType have same structure as galaxy versions. Multiple calling functions are virtual and cross deepskyobjects, nebula, galaxy, globular and opencluster.  Keep present code and you can decide on preferred code and implement across all DSOs to maintain uniformity in the future.

Reply:
done